.travis.yml

```diff
@@ -16,8 +16,10 @@ env:
   # run tests/cpp
   - TASK=cpp_test
   # run tests/python
-  - TASK=python_test
```

Comment:
I think its better to keep python tests on mac until we have our own mac servers.